nengo/builder/learning_rules.py

```diff
@@ -86,10 +97,30 @@ def make_step(self, signals, dt, rng):
         theta = signals[self.theta]
         delta = signals[self.delta]
         alpha = self.learning_rate * dt
+        period = (1 if self.apply_every is None else
```

Comment:
Is there any difference using the ternary vs. normal or is this purely a preference/readability thing?

Maybe @tbekolay has an opinion on refactoring this? My understanding is that the operator must return instructions that get applied to the signals. Am I able to return something wrapped in a function?